hedera-node/src/test/java/com/hedera/services/tokens/HederaTokenStoreTest.java

```diff
@@ -1226,8 +1223,8 @@ public void burningRejectsInvalidNewSupply() {
 		long halfwayToOverflow = (1L << 62) / 2;
```

Comment:
I think that you should edit this variable as-well. 
1) The decimal is not actually of any importance. 
2) its not actually halfway

The test is passing because you are passing burn amount which is supply + 1